2-Pricing/src/components/Cart.tsx

```diff
@@ -0,0 +1,94 @@
+import type { Cart as ICart } from "../App";
+import { Backdrop } from "./Backdrop";
+import { Button } from "./Button";
+import { Subscription, subscriptions } from "../constants";
+import { CheckoutRow } from "./CheckoutRow";
+
+export type SubscriptionKey = keyof typeof subscriptions;
+
+interface CartProps {
+  isMonthlyView: boolean;
+  inCart: ICart;
+  setIsCartOpen: (cartIsVisible: boolean) => void;
```

Comment:
In these cases, I think it makes sense for the parameter to have the same name as the one used in the function name:

```suggestion
  setIsCartOpen: (isCartOpen: boolean) => void;
```